pkg/api/mlflow/service/run/validators.go

```diff
@@ -0,0 +1,123 @@
+package run
+
+import (
+	"github.com/G-Research/fasttrack/pkg/api/mlflow/api"
+	"github.com/G-Research/fasttrack/pkg/api/mlflow/api/request"
+)
+
+const (
+	MaxResultsPerPage = 1000000
+)
+
+// AllowedViewTypeList supported list of ViewType.
+var (
+	AllowedViewTypeList = map[request.ViewType]struct{}{
+		"":                          {},
+		request.ViewTypeAll:         {},
+		request.ViewTypeActiveOnly:  {},
+		request.ViewTypeDeletedOnly: {},
+	}
+)
+
+// ValidateUpdateRunRequest validates `POST /mlflow/runs/update` request.
+func ValidateUpdateRunRequest(req *request.UpdateRunRequest) error {
+	if req.RunID == "" && req.RunUUID == "" {
+		return api.NewInvalidParameterValueError("Missing value for required parameter 'run_id'")
+	}
+	return nil
+}
+
+// ValidateGetRunRequest validates `GET /mlflow/runs/get` request.
+func ValidateGetRunRequest(req *request.GetRunRequest) error {
+	if req.RunID == "" && req.RunUUID == "" {
+		return api.NewInvalidParameterValueError("Missing value for required parameter 'run_id'")
+	}
+	return nil
+}
+
+// ValidateDeleteRunRequest validates `POST /mlflow/runs/delete` request.
+func ValidateDeleteRunRequest(req *request.DeleteRunRequest) error {
+	// TODO:DSuhinin do we need to check here req.RunUUID?
```

Comment:
no need to check it + same for all the `TODO` comments below

Reply:
ok, removed.